pkgs/development/compilers/odin/default.nix

```diff
@@ -37,21 +47,22 @@ in stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/bin
     cp odin $out/bin/odin
-    cp -r core $out/bin/core
-
-    wrapProgram $out/bin/odin --prefix PATH : ${lib.makeBinPath (with llvmPackages; [
-      bintools
-      llvm
-      clang
-      lld
-    ])}
+
+    wrapProgram $out/bin/odin \
+      --prefix PATH : ${lib.makeBinPath (with llvmPackages; [
+        bintools
+        llvm
+        clang
+        lld
+      ])} \
+      --set-default ODIN_ROOT ${src}
```

Comment:
You should copy over the directories into `$out/share`, and set the path here to `$out/share`.

Doing it the way you do now will cause the odin binary to depend on its source, which will bloat people's nix stores